src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -152,12 +152,12 @@ protected override void CreateVersions()
             // first add all the available versions to be updated
             foreach (var version in allVersionsAllowed)
             {
-                var installed = version.Equals(installedVersion);
+                var installed = version.version.Equals(installedVersion);
```

Comment:
version.version = boo :)

Maybe rename the variable?

Reply:
yeah, definitely not the best, but fixing all the titling issues in this code would need a lot of variable renames